editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -78,8 +76,68 @@ function MapAndLabelComponent(props: Props) {
             </InputRow>
           </InputGroup>
         </ModalSectionContent>
-        <ModalSectionContent title="Map formatting">
+        <ModalSectionContent title="Map options">
           <InputGroup>
+            <Box mb={2}>
+              <FormControl component="fieldset">
+                <InputLabel label="Basemap">
+                  <RadioGroup
+                    defaultValue="OSVectorTile"
+                    value={formik.values.basemap}
+                  >
+                    {[
+                      {
+                        id: "OSVectorTile",
+                        title: "Ordnance Survey Vector Tiles",
+                      },
+                      {
+                        id: "MapboxSatellite",
+                        title: "Mapbox Satellite imagery",
+                      },
```

Comment:
A nice future improvement here would be to add small static thumbnails of vector tile versus satellite imagery - I don't necessarily expect these names alone to be meaningful to editors! 

I've also intentionally excluded the "OSRaster" & "OSM" options here even though they're valid configs in the map repo - we don't have PlanX use cases for them!